generators/generator-constants.js

```diff
@@ -69,13 +69,7 @@ const DOCKER_KEYCLOAK = 'jboss/keycloak:11.0.1'; // The version should match the
 const DOCKER_ELASTICSEARCH = 'docker.elastic.co/elasticsearch/elasticsearch:7.6.2'; // The version should be coherent with the one from spring-data-elasticsearch project
 const DOCKER_KAFKA = `confluentinc/cp-kafka:${KAFKA_VERSION}`;
 const DOCKER_ZOOKEEPER = `confluentinc/cp-zookeeper:${KAFKA_VERSION}`;
-const DOCKER_SONAR = 'sonarqube:8.4.2-community';
-const DOCKER_JHIPSTER_CONSOLE = 'jhipster/jhipster-console:v4.1.0';
-const DOCKER_JHIPSTER_CURATOR = 'jhipster/jhipster-curator:v4.1.0';
-const DOCKER_JHIPSTER_ELASTICSEARCH = 'jhipster/jhipster-elasticsearch:v4.1.0';
-const DOCKER_JHIPSTER_LOGSTASH = 'jhipster/jhipster-logstash:v4.1.0';
-const DOCKER_JHIPSTER_IMPORT_DASHBOARDS = 'jhipster/jhipster-import-dashboards:v4.1.0';
-const DOCKER_JHIPSTER_ZIPKIN = 'jhipster/jhipster-zipkin:v4.1.0';
+const DOCKER_SONAR = 'sonarqube:8.4.1-community';
```

Comment:
bad change, you downgrade the version of sonar here

Reply:
just fixed it